src/modules/format-utils.js

```diff
@@ -204,7 +204,62 @@ function timeLeft(in_date) {
     return `in ${oxfordStringifyValues(labels, 'and')}`;
 }
 
+
+/**
+ * Unescapes HTML entities (&#\d+;) only
+ * @param {String} str String containing HTML numeric entities
+ * @returns {String} An unescaped string
+ */
+function unescapeEntities(str) {
+    if (typeof str === 'object') {
+        try {
+            str = str.valueOf();
+        }
+        catch ( error ) {
+            error.message = 'Utils: tried turning argument into a string, unsuccessful' + error.message;
+            throw error;
+        }
+    }
+    if (typeof str !== 'string')
+        throw new TypeError(`Utils: bad input for string to unescape: Expected string, got ${typeof str}`);
+    return str.replace(/&#(\d+);/gi, function(match, numStr) {
+        const num = parseInt(numStr, 10);
+        return String.fromCharCode(num);
+    });
+}
+
+
+/**
+ * Checks if a string is a valid URL
+ * @param {String} str String to be tested
+ * @returns {Boolean} Whether it's valid
+ */
+function isValidURL(str) {
+    if (typeof str === 'object') {
```

Comment:
Make a shared method for this coercion to string, since it is duplicated between at least two methods. Also, the coverage report indicates we don't hit the catch clause of either of these uses - what inputs would we get that would trigger them?

Reply:
It'd have to be a weird object that can't be coerced into a string through valueOf(). So probably something that explicitly disables that method - which may never exist or be too edge-case.

Reply:
I wonder if it's even with having in that case - these methods are only for stuff we are working with firsthand, such as strings, containers, or json objects from http calls. If we were eg a library some other application could use, that would be different